src/components/constructor-form/constructor-form.js

```diff
@@ -0,0 +1,347 @@
+    handleImageLoad(e, (event) => {
+      setFieldValue('image', event.target.result);
+      setConstructorImg(event.target.result);
+    });
+  };
+
+  const textField = (inputValue, inputLabel) => (<TextField
+    data-cy={inputValue}
+    id={inputValue}
+    className={styles.textField}
+    variant='outlined'
+    label={inputLabel}
+    value={inputValue}
+    onChange={handleChange}
+    error={touched.inputValue && !!errors.inputValue}
+  />)
+
+  return (
+    <div>
+      <form onSubmit={handleSubmit}>
+        <CheckboxOptions options={checkboxes} />
+        <Grid item xs={12}>
+          <Paper className={styles.constructorItemUpdate}>
+            <div>
+              <span className={styles.imageUpload}>
+                {config.labels.model.constructorPhoto}
+              </span>
+              <div className={styles.imageUploadAvatar}>
+                <ImageUploadContainer handler={handleLoadConstructorImage} />
+                {constructorImg && (
+                  <Avatar src={constructorImg} variant='rounded' className={styles.avatar}>
+                    <Image />
+                  </Avatar>
+                )}
+                {touched.image &&
+                errors.image && (
+                  <div className={styles.inputError}>
+                    {errors.image}
+                  </div>
+                )}
+              </div>
+            </div>
+            {textField(values.basePrice, config.labels.model.basePrice)}
+            {touched.basePrice && errors.basePrice && (
+              <div className={styles.inputError}>{errors.basePrice}</div>
+            )}
+            <FormControl variant='outlined' className={styles.textField}>
+              <InputLabel id='multiple-checkbox-label'>
+                {config.labels.model.material}
+              </InputLabel>
+              <Select
+                labelId='multiple-checkbox-label'
+                id='multiple-checkbox'
+                onChange={handleMaterial}
+                input={<Input />}
+                defaultValue={values.material}
+                MenuProps={MenuProps}
+              >
+                {list.map((cat) => (
+                  <MenuItem value={cat._id} key={cat._id}>
+                    <div className={styles.selectBox}>
+                      <ColorCircle size={SMALL_CIRCLE} color={cat.color.colorHex} />
+                      <span> {cat.name[0].value}</span>
+                    </div>
+                  </MenuItem>
+                ))}
+              </Select>
+              {touched.material && errors.material && (
+                <div className={styles.inputError}>{errors.material}</div>
+              )}
+            </FormControl>
+
+          </Paper>
+        </Grid>
+        <AppBar position='static'>
+          <Tabs
+            className={styles.tabs}
+            value={tabsValue}
+            onChange={handleTabsChange}
+            aria-label='simple tabs example'
```

Comment:
This area-label name is unsuitable in this case

Reply:
Fixed